gammapy/data/tests/test_event_list.py

```diff
@@ -52,6 +52,9 @@ def test_region(self):
     def test_peek(self):
         self.events.peek()
 
+    def test_plot_theta2_distribution(self):
+        self.plot_theta2_disitribution()
```

Comment:
tests do not pass in travis because this typing error
plot_theta2_disitribution -> plot_theta2_distribution

Reply:
Also, I think you have to call `self.events.plot_theta2_distribution` here.
To run the test locally you can use:
```
python setup.py test -V -t gammapy/data/tests/test_event_list.py
```

Reply:
Thanks I didn't have the right command line.